tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -1424,15 +1424,15 @@ def test_measurement_rating(self):
         )
 
     def test_team_key_transactions_my_teams(self):
-        for idx, (transaction, value) in enumerate(
```

Comment:
is change related to this problem?

Reply:
Nope, by mistake I refactored also that name. It was done also to unify.